plugins/lookup/passwordstore.py

```diff
@@ -304,6 +347,10 @@ def run(self, terms, variables, **kwargs):
                 else:
                     result.append(self.get_passresult())
             else:                     # password does not exist
-                if self.paramvals['create']:
+                if self.paramvals['missing'] == 'create':
                     result.append(self.generate_password())
+
+        if not result:
+            result.append(None)
+
```

Comment:
```suggestion
                    result.append(self.generate_password())
                else:
                    result.append(None)

```
There should really be one entry in `result` per term.

Reply:
Ah, sure, because multiple passname might be missing, so we want multiple `None`s to be in the result list. Makes sense.